<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="LAPTOP-KQRS1J5O" failures="0" tests="2" name="parallelTest" time="12.817" errors="0" timestamp="2024-08-23T18:20:53 IST">
  <testcase classname="testScript.LoginPage_Test" name="verify_User_is_Able_To_Login_Using_Valid_Credentials" time="1.548"/>
  <testcase classname="testScript.NewAdmin_Test" name="verify_User_is_Able_To_Add_NewUser" time="6.031"/>
  <testcase classname="testScript.LoginPage_Test" name="verify_User_is_Able_to_Login_Using_Valid_Username_invalid_Password" time="0.0">
    <ignored/>
  </testcase> <!-- verify_User_is_Able_to_Login_Using_Valid_Username_invalid_Password -->
  <testcase classname="testScript.LoginPage_Test" name="verify_User_is_Able_To_Login_Using_invalid_Username_Valid_Password" time="0.0">
    <ignored/>
  </testcase> <!-- verify_User_is_Able_To_Login_Using_invalid_Username_Valid_Password -->
  <testcase classname="testScript.LoginPage_Test" name="verify_User_is_Able_to_Login_Using_Invalid_Username_Invalid_Password" time="0.0">
    <ignored/>
  </testcase> <!-- verify_User_is_Able_to_Login_Using_Invalid_Username_Invalid_Password -->
</testsuite> <!-- parallelTest -->
